src/parse.c

```diff
@@ -998,6 +1001,19 @@ void ParseWindowStyle(const TokenNode *tp)
 {
    const TokenNode *np;
 
+   const char *temp;
+   temp = FindAttribute(tp->attributes, MACHINENAME_ATTRIBUTE);
+   if(temp){
+        settings.showMachineName = !strcmp(temp,TRUE_VALUE);
+   }
+
+   temp = FindAttribute(tp->attributes, MN_DELIMITERS_ATTRIBUTE);
+   if(temp && strlen(temp)>=2) {
+      char *delims = calloc(strlen(temp)+1,sizeof(char));
+      sprintf(delims,"%s",temp);
```

Comment:
I think I would actually just declare `settings.machineNameDelimiters` as `char machineNameDelimiters[2];` and just memcpy two characters here